api/models.py

```diff
@@ -11,6 +12,8 @@ def __str__(self):
 class Entity(BaseClass):
     location = models.CharField(max_length=120)
     user = models.ForeignKey('auth.User')
+    email = models.EmailField(max_length=60)
+    phone = models.CharField(max_length=9, validators=[phoneregex])
```

Comment:
I like what you did here but let's not create a new file for validation. Keep in mind that no one else in our DB will have a phone number, just put the validator in the same file, something like:
```
from django.core.validators import RegexValidator

...
phone_regex = RegexValidator(regex=r'^[0-9]+', message="Enter valid phone number")
phone_number = models.CharField(validators=[phone_regex], blank=True, null=True)
```
Something similar can be found [here](https://stackoverflow.com/questions/19130942/whats-the-best-way-to-store-phone-number-in-django-models)

Reply:
Ok, I'll change that in a sec. The reason to create a new file for validations is that it can also be used in views and not only in models.
I thougth it could be interesting to declare that in a explicit way, but it's ok for me to keep validations inside where they are used :)

Reply:
The same validator can be used in both Model fields and Form field. So, for example, we can use the phone_validator in the model itself and in the form used to create it.

I don't know if we should use it in both places or just in one of them, I'm just giving the info :)

Reply:
You are both right, but let's build for the present. Currently we do not have forms or views that use phone validation (or any validation) so there is no need to call the validation file anywhere but in the models.
Double validating it's a good strategy but @Nosk14 do you remember that you can [create forms from models](https://docs.djangoproject.com/en/1.11/topics/forms/modelforms/), right? In this case I'm not sure if you will need a validation in forms.py or views.py since the forms work at the same level and you already defined the validation rules in the models :)